### PR TITLE
Update dark-sites.config

### PR DESCRIPTION
**Pull request by gordcodes**
add ign.com, hulu.com, netflix.com

### CODE REVIEW

**Review by gordcodes — COMMENTED** [review]
Add youtube.com

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -328,6 +328,7 @@ hoppscotch.io
 hostedtalk.net
 hotstar.com
 htmlpasta.com
+hulu.com
```
Comment (Gusted) [inline src/config/dark-sites.config:331]: image
Doesn't seem that dark to me.

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -340,6 +341,7 @@ ibookpile.net
 identinator.com
 ieuan.tech
 ieuan.xyz
+ign.com
```
Comment (Gusted) [inline src/config/dark-sites.config:344]: Not that dark either
image

File: src/config/dark-sites.config [hunk 3]
```diff
@@ -444,6 +446,7 @@ nanhu.ca
 nchristopher.me
 neal.fun/size-of-space
 neovim.io
+netflix.com
```
Comment (Gusted) [inline src/config/dark-sites.config:449]: Already been listed under www.netflix.com

File: src/config/dark-sites.config [hunk 4]
```diff
@@ -771,6 +774,7 @@ yandexdataschool.ru/$
 yasio.pl
 yohoho.io
 you.dj
+youtube.com
```
Comment (Gusted) [inline src/config/dark-sites.config:777]: Won't need to open youtube to know this one isn't dark either.

File: src/config/dark-sites.config (hunk 1 above)
Comment (gordcodes) [reply src/config/dark-sites.config:331]: If you have a hulu account and log in the entire site is dark
image

File: src/config/dark-sites.config (hunk 2 above)
Comment (gordcodes) [reply src/config/dark-sites.config:344]: I guess IGN International hasn't rolled out dark mode based on theme yet
image

File: src/config/dark-sites.config (hunk 4 above)
Comment (gordcodes) [reply src/config/dark-sites.config:777]: Looks like Youtube US may only have themes and hasn't rolled it out internationally yet also. And this is with the dark mode add-on disabled.
image

**Comment by gordcodes** [comment]
@Gusted  can you actually check youtube? Do you actually not have a dark mode in Europe?

**Comment by Gusted** [comment]
>  Do you actually not have a dark mode in Europe?

We have an dark-mode, but just not by default, which is a must to get an site into this list.

**Pull request closed without merging**